assets/js/blocks/stock-filter/utils.ts

```diff
@@ -23,8 +23,9 @@ export const getActiveFilters = (
 		? params.split( ',' )
 		: ( params as string[] );
 
-	return Object.keys( filters ).filter( ( filter ) =>
```

Comment:
Here, the filters variable, which is a predefined collection of stock statuses coming from the server used to define the order of status since that's what the filter was applied on. 

I instead put the filter on `parsedParams`, which are the stock statuses selected by the user. This variable also has them in the correct order.